src/seedu/addressbook/data/address/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.address;
+
+public class Block {
+	private final String value;
+	
+	public Block(String value){
+		this.value = value;
+	}
+	
+	@Override
+	public String toString(){
+		return value;
+	}
+	
+	@Override
+    public boolean equals(Object other) {
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.